deploy/role.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: infinispan-operator
+  creationTimestamp: null
```

Comment:
This field is not needed

Reply:
Good spot. I think this was added when the sdk added the batch permissions

Reply:
Yes, looks like very old SDK bug